src/app/components/rpc-transaction-request/transaction-recipients.layout.tsx

```diff
@@ -1,25 +1,38 @@
+import { SharedComponentsSelectors } from '@tests/selectors/shared-component.selectors';
 import { HStack, styled } from 'leather-styles/jsx';
 
-import { AddressDisplayer, Approver, BtcAvatarIcon, ItemLayout, UserIcon } from '@leather.io/ui';
-import { formatDustUsdAmounts, formatMoneyPadded, i18nFormatCurrency } from '@leather.io/utils';
+import type { Money } from '@leather.io/models';
+import { AddressDisplayer, Approver, ItemLayout, UserIcon } from '@leather.io/ui';
+import {
+  formatDustUsdAmounts,
+  formatMoney,
+  formatMoneyPadded,
+  i18nFormatCurrency,
+} from '@leather.io/utils';
 
 import type { TransferRecipient } from '@shared/models/form.model';
 
-import { useConvertCryptoCurrencyToFiatAmount } from '@app/common/hooks/use-convert-to-fiat-amount';
 import { IconWrapper } from '@app/components/icon-wrapper';
 import { Divider } from '@app/components/layout/divider';
 
-interface TransactionRecipientsProps {
+interface TransactionRecipientsLayoutProps {
+  avatar: React.ReactNode;
+  caption: string;
   recipients: TransferRecipient[];
+  title: string;
+  convertToFiatAmount(value: Money): Money;
 }
-// TODO: Refactor to decouple from Bitcoin
-export function TransactionRecipients({ recipients }: TransactionRecipientsProps) {
-  const convertToFiatAmount = useConvertCryptoCurrencyToFiatAmount('BTC');
-
+export function TransactionRecipientsLayout({
+  avatar,
+  caption,
+  recipients,
+  title,
+  convertToFiatAmount,
+}: TransactionRecipientsLayoutProps) {
   return recipients.map(({ address, amount }) => {
     const fiatAmount = convertToFiatAmount(amount);
 
-    const titleRight = formatMoneyPadded(amount);
+    const titleRight = amount.symbol === 'BTC' ? formatMoneyPadded(amount) : formatMoney(amount);
```

Comment:
I have never really understood this, why do we use `formatMoneyPadded` for Bitcoin? It always creates problems with shared components.

Reply:
I don't ever see it shown this way in designs, so I'm going to remove it. This is not how BTC values are shown in Figma...

`formatMoneyPadded`
![Screenshot 2025-04-29 at 3 58 21 PM](https://github.com/user-attachments/assets/9dee5dfd-6cf7-4d8e-a802-324f863531a3)

`formatMoney`
![Screenshot 2025-04-29 at 4 00 46 PM](https://github.com/user-attachments/assets/66f0f1f4-33b9-4e88-bb31-a9ad2a415dbd)

Reply:
When numbers are displayed on top of each other, qualifying the additional 0s helps reference the exact digit. In our tx summary for instance, we should pad the zeros

Reply:
Why we would only do it for bitcoin, I don't understand, we should always do it imo